fv_be/app/models/characters.py

```diff
@@ -30,9 +31,13 @@ class Meta:
     notes = models.TextField(blank=True)
 
     # from fvcharacter:related_words
-    # TODO: add this as a foreign key to the dictionary entry model once it's created
-    # related_dictionary_entries = models.ForeignKey(
-    #     DictionaryEntry, on_delete=models.SET_NULL, blank=True, null=True, related_name="characters")
+    related_dictionary_entries = models.ForeignKey(
```

Comment:
I think this actually needs to be a many-to-many relation, right?

Reply:
Changed from an FK to a many to many field, I do not think we need a "through" class for this relation but let me know if we do.